### ISSUE TITLE
Rails/SkipsModelValidations should not validate PORO

### ISSUE THREAD

**Issue by wpolicarpo** [bbatsov/rubocop#5183]
I have a class (PORO) containing a method called `#touch` and Rubocop flags this method as a `Rails/SkipsModelValidations` offense.

## Expected behavior

Rubocop should not flag all `#touch` methods as bad.

## Actual behavior

Rubocop is flagging `MyClass#touch` method as bad, even though `MyClass` is not an `ActiveRecord` or `ActiveModel` class.

### Code
```ruby
# my_class.rb
class MyClass
  def touch
  end
end

MyClass.new.touch
```

### Run
```bash
rubocop -D my_class.rb
Inspecting 1 file
C

Offenses:

a.rb:8:10: C: Rails/SkipsModelValidations: Avoid using touch because it skips validations.
instance.touch
         ^^^^^

1 file inspected, 1 offense detected
```

## RuboCop version

```
$ rubocop -V
0.51.0 (using Parser 2.4.0.2, running on ruby 2.4.2 x86_64-darwin16)
```

**Comment by mikegee** [comment]
Unfortunately, Rubocop can't know the type of the `instance` object in `instance.touch`.

This cop can be disabled globally for your application or inline with:

```ruby
instance.touch # rubocop:disable Rails/SkipsModelValidations
```
